lib/libfrr.c

```diff
@@ -968,6 +968,10 @@ static void frr_config_read_in(struct thread *t)
 {
 	hook_call(frr_config_pre, master);
```

Comment:
hmm - it looks like there are already hook function callbacks in this path - could you just use those?

Reply:
Updated